beacon-chain/core/validators/validator_test.go

```diff
@@ -217,6 +217,7 @@ func TestSlashValidator_OK(t *testing.T) {
 }
 
 func TestSlashValidator_Electra(t *testing.T) {
+	helpers.ClearCache()
```

Comment:
Running `TestSlashValidator_Electra` by itself passes but running the entire package will fail because previous test uses cache. This wasn't an issue before but now becomes an issue because shuffling is different after Electra